### PR TITLE
View Other

### PR DESCRIPTION
**Pull request by braedonwatkins**

### CODE REVIEW

File: src/pages/api/user/getUserId.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    req: NextApiRequest,
+    res: NextApiResponse<GetUserResponseType>
+) {
+
+    // Find user information.
+    const user = await prisma.user.findMany({
+        select: {
+            id: true,
+        }
+    })
```
Comment (Gian-Alvarez) [inline src/pages/api/user/getUserId.ts:28]: Is email supposed to be used here to get a specific ID or are you querying for all ID's.

**Review by Gian-Alvarez — APPROVED** [review]

File: src/pages/api/user/getUserId.ts (hunk 1 above)
Comment (xTye) [reply src/pages/api/user/getUserId.ts:28]: He is querying all ids, originally meant for GetStaticPaths

File: src/pages/users/[userid].tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    }
+    
+    // fallback case so next isn't angry
+    return {
+      props: { propTours: [], userid: null }
+    }
+  };
+
+
+export default NewPage
```
Comment (xTye) [inline src/pages/users/[userid].tsx:153]: add new line

File: src/pages/users/[userid].tsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+import Router, { useRouter } from 'next/router';
+import { Tour, User } from '@prisma/client';
+import { getToken } from 'next-auth/jwt';
+import { prisma } from "../../lib/prisma";
+import { CreateTourResponseType } from '../api/tour';
+import { useEffect, useState } from 'react';
+import Error from 'next/error';
+
+
+const NewPage: NextPage = ({ propTours, userid }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
```
Comment (xTye) [inline src/pages/users/[userid].tsx:14]: Rename the page to ViewOtherPage

File: src/pages/users/[userid].tsx [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    const router = useRouter();
+    const { data: session, status } = useSession();
+    const [tours, setTours] = useState(propTours);
+    const [query, setQuery] = useState("");
+    const [changing, setChanging] = useState(false);
+    let timer: NodeJS.Timeout;
+  
+    const queryTours = () => {
```
Comment (xTye) [inline src/pages/users/[userid].tsx:22]: Eventually, we will make this card search functionality its own component, but this is good for now.

**Review by xTye — APPROVED** [review]
Looks good, implement comments and then merge

**Pull request merged**